rust-connectors/sources/mqtt/README.md

```diff
@@ -14,12 +14,16 @@ MQTT V3.1.1 and V5
 
 Controls the Source connector
 
-| Option     | default  | type   | description |
-| :---       | :---     | :---   | :----       |
-| timeout    | 60       | u64    | mqtt broker connect timeout in seconds |
-| mqtt_url   | -        | String | mqtt broker Host |
-| mqtt_topic | -        | String | mqtt topic to subscribe and source events from |
-| client_id  | UUID V4  | String | mqtt client ID |
+| Option        | default  | type   | description                                             |
+| :---          | :---     | :---   | :----                                                   |
+| timeout       | 60       | u64    | mqtt broker connect timeout in seconds                  |
+| mqtt_url      | -        | String | mqtt broker Host                                        |
```

Comment:
mqtt_port, mqtt_tls should be incorporated in mqtt_url not separately perhaps? mqtt url should support differnet port / tls schemes perhaps? I know it's separate issue on #164 but?

Reply:
or should we just perhaps deprecate and change it to host as to clear intent?